lib/database/player-game.php

```diff
@@ -312,7 +314,7 @@ function getGameRecentPlayers($gameID, $maximum_results = 0): array
 
     $query = "SELECT ua.ID as UserID, ua.User, ua.RichPresenceMsgDate AS Date, ua.RichPresenceMsg AS Activity
               FROM UserAccounts AS ua
-              WHERE ua.LastGameID = $gameID AND ua.Permissions >= 0
+              WHERE ua.LastGameID = $gameID AND ua.Permissions >= " . Permissions::Registered . "
```

Comment:
0 is `Permissions::Unregistered`. Was this change intentional?

Reply:
Oh, no 🙈  Thanks!